src/Azure.Functions.ArtifactAssembler/Program.cs

```diff
@@ -11,8 +13,8 @@ static async Task<int> Main(string[] args)
             {
                 var currentWorkingDirectory = Environment.CurrentDirectory;
                 var artifactAssembler = new ArtifactAssembler(currentWorkingDirectory);
-                await artifactAssembler.AssembleArtifactsAsync();
 
+                await artifactAssembler.AssembleArtifactsAsync();
```

Comment:
Not sure I understand why a change was made here? Looks the same to me - can we undo?